nengo_dl/simulator.py

```diff
@@ -1160,18 +1160,16 @@ def loss(self, *args, **kwargs):
 
     @require_open
     @with_self
-    def save_params(self, path, include_non_trainable=False):
+    def save_params(self, path, include_state=False, include_non_trainable=None):
         """
         Save network parameters to the given ``path``.
 
         Parameters
         ----------
         path : str
             Filepath of parameter output file.
-        include_non_trainable : bool
-            If True (default False) also save information representing
-            non-trainable parameters of the network (this includes the internal
-            simulation state).
+        include_state : bool
+            If True (default False) also save the internal simulation state.
```

Comment:
I would opt for still having a line for `include_non_trainable`, that basically says it's deprecated and equivalent to `include_state`.

Reply:
Looking at `nengo.Simulator.trange` and the switch in parameters there, we don't have documentation for both the old and new names, but we do have a "version changed" tag, so I've opted for that instead.